play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

```diff
@@ -30,7 +30,7 @@ Duration: 	Free Action
 Range:		Self
 Effect:		Increase move speed to 12m per action
 and get a +30 to accrobatics checks. Effect persists for 1 turn. While 
-in a medical emergency, the Doc suffers a +28% chance to miss, 'cause
+in a medical emergency, the Doc suffers a +2.5 chance to miss, 'cause
```

Comment:
I've been keeping the miss chances to a whole number value unless it's an attachment. If you want to use a decimal number you'll need to define the rules for rounding that number (rounding up/down/w.e. after accounting for all other factors).
The pickle I've been dealing with is, "when do we make the player round the numbers?" To make it simple, I've designed it thus far that the player rounds (drops the remainder) miss chance values for weapons when they go to write the values down in their 3x3 table.
So in summary, all effects in the middle of play have to either define how they are supposed to be rounded, or they should be whole number values.

Skill checks are a separate beast, of course, since they aren't d10 scale.

Reply:
Can't perception mod be a half number? Or am I miss-remembering that?

Reply:
It can be a half number. But what happens when it's a whole number, such that your miss chance leaves a half number?

Reply:
Then you already wrote the rule that it rounds down? I think dropping it to 2.0 would be too weak, and 3.0 would be too strong.

Reply:
Oops, just saw this. The rounding rules apply before abilities come in, I don't have any abilities that use halves... (I hope?)